README.md

```diff
@@ -39,15 +39,15 @@ Install instructions
 
 Installing projman_filler can be done in the following way.
 
-Please note that projman_filler uses python 3.6, ensure this is first installed.
+Please note that projman_filler uses python 3.10, ensure this is first installed.
 
  * Download the projman_filler delivery
  * Install projman_filler with pip as normal, i.e. `pip install <path to source>`
 
 
 Development and testing
 -----------------------
-Set up a Python environment using the latest version of Python 3.6 (3.6.12 as of this writing).
+Set up a Python environment using the latest version of Python 3.10 (3.10.1 as of this writing).
```

Comment:
3.10.15 is the latest version of python 3.10? Either way, I don't think we have to mention the latest version as of writing, so I think you can remove the whole parenthesis